src/taco-cli/cli/installReqs.ts

```diff
@@ -37,10 +37,10 @@ import loggerHelper = tacoUtils.LoggerHelper;
  * Handles "taco install-dependencies"
  */
 class InstallReqs extends commands.TacoCommandBase {
-    private static KnownOptions: Nopt.FlagTypeMap = { };
-    private static PlatformsFolderName: string = "platforms";
-    private static PathToCheckReqs: string = path.join("cordova", "lib", "check_reqs.js");
-    private static MinPlatformVersions: { [id: string]: string } = {
+    private static KNOWN_OPTIONS: Nopt.FlagTypeMap = { };
+    private static PLATFORMS_FOLDER_NAME: string = "platforms";
+    private static CHECK_REQS_PATH: string = path.join("cordova", "lib", "check_reqs.js");
+    private static minPlatformVersions: { [id: string]: string } = {
```

Comment:
Why is "minPlatformVersions" not made into "MIN_PLATFORM_VERSIONS" like the others?